.github/workflows/pkgdown.yml

```diff
@@ -3,6 +3,7 @@ on:
     branches:
       - main
       - master
+      - vign
```

Comment:
Generally want to remove this before sending PR

Reply:
Just removed with last commit 2444895
 Good tip for future PR, such a handy way to test!